Formula/rust.rb

```diff
@@ -33,6 +39,7 @@ class Rust < Formula
   end
 
   option "with-llvm", "Build with brewed LLVM. By default, Rust's LLVM will be used."
+  option "with-racer", "Builds racer code completion tool, and retains rust sources."
```

Comment:
Also worth noting in case it slipped your mind: https://github.com/Homebrew/homebrew-core/pull/6825#issuecomment-260114533